packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -1,143 +1,64 @@
-
 %global gem_name smart_proxy_salt
 %global plugin_name salt
 
-%global foreman_proxy_min_version 2.5
-%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
-%global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_sharedstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
-
-%global salt_config_dir %{_root_sysconfdir}/salt
-%global salt_proxy_runners_dir %{foreman_proxy_dir}/salt/runners
-%global salt_proxy_reactors_dir %{foreman_proxy_dir}/salt/reactors
-%global salt_state_grains_dir %{foreman_proxy_statedir}/salt/grains
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
 
-Summary: SaltStack support for Foreman Smart-Proxy
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 5.0.1
+Name: rubygem-%{gem_name}
+Version: 5.1.0
 Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
+Summary: SaltStack Plug-In for Foreman's Smart Proxy
 License: GPLv3
 URL: https://github.com/theforeman/smart_proxy_salt
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: salt-master
-%if %{with python2}
-Requires: python
-BuildRequires: python2-rpm-macros
-%endif
-%if %{with python3}
-Requires: python3
-BuildRequires: python3-rpm-macros
-%endif
-
 # start specfile generated dependencies
 Requires: foreman-proxy >= %{foreman_proxy_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(smart_proxy_dynflow) >= 0.5.0
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.7
+Requires: ruby < 4
+BuildRequires: ruby >= 2.7
+BuildRequires: ruby < 4
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 Provides: foreman-proxy-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
-%{?scl:Obsoletes: rubygem-%{gem_name} < %{version}-%{release}}
-
 %description
-SaltStack Plug-In for Foreman's Smart Proxy.
+This plug-in adds support for Salt to Foreman's Smart Proxy.
 
 
 %package doc
 Summary: Documentation for %{name}
-Group: Documentation
 Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
-%{?scl:Obsoletes: rubygem-%{gem_name}-doc < %{version}-%{release}}
-
 %description doc
 Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
-
-%if %{with python2} && 0%{?rhel} >= 8
-sed -i -e '1s|^#!.*$|#!%{__python2}|' sbin/upload-salt-reports
-%endif
-
-%if %{with python3} && 0%{?rhel} >= 8
-sed -i -e '1s|^#!.*$|#!%{__python3}|' sbin/upload-salt-reports
```

Comment:
the upload/salt-reports should then have python3 in the file?

Reply:
I think this is already good: https://github.com/theforeman/smart_proxy_salt/blob/924fcfb7a3739cd4b62cd01c55a4f618d98e8db1/sbin/upload-salt-reports#L1

```suggestion
```